application/src/test/java/uk/nhs/hee/tis/revalidation/service/DoctorsForDBServiceTest.java

```diff
@@ -21,7 +21,6 @@
 
 package uk.nhs.hee.tis.revalidation.service;
 
-import static java.time.LocalDate.now;
```

Comment:
Is it intentional to get rid of the static import?

Reply:
Yes.  I qualified all of the calls.
I focused only on this static import because the class uses both `LocalDate` & `LocalDateTime`.